source/educators/references/course_development/modify_outline_settings.rst

```diff
@@ -24,4 +24,26 @@ circled for a section, a subsection, and two units.
  :alt: Configure icons in the course outline.
  :width: 500
 
-For more information, see the links above.
\ No newline at end of file
+For more information, see the links above.
+
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```

Comment:
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Developing Your Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for Open edX Course Teams <Resources for edx org Course Teams>` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

Reply:
Resolved spacing issues in See Also section.